testsuite/tests/tool-toplevel/known-bugs/broken_rec_in_show.ml

```diff
@@ -41,7 +41,38 @@ module rec M : sig type t val x : M.t end = struct type t = int let x = 0 end;;
 module rec M : sig type t val x : M.t end
 |}];;
 #show_module M;;
-(* this output is INCORRECT, it should use 'rec' *)
```

Comment:
There is an odditiy on lines 35-36 (github wouldn't let me comment there), with an empty `expect` block.